xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -36,7 +36,7 @@ using namespace PVR;
 using namespace EPG;
 
 CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) :
-  CPVRChannelGroup(bRadio, bRadio ? PVR_INTERNAL_GROUP_ID_RADIO : PVR_INTERNAL_GROUP_ID_TV, g_localizeStrings.Get(19287)),
+  CPVRChannelGroup(bRadio, -1, g_localizeStrings.Get(19287)),
```

Comment:
I would use the default ctor of `CPVRChannelGroup` and set the necessary members in the body as -1 is the default for `m_iGroupId`.